sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Security/Ucfg/UniversalControlFlowGraphBuilder_Instructions.cs

```diff
@@ -60,7 +60,7 @@ public void Foo(string s)
 }";
             var ucfg = GetUcfgForMethod(code, "Foo");
 
-            ucfg.BasicBlocks.Should().HaveCount(1);
+            ucfg.BasicBlocks.Should().HaveCount(2);
```

Comment:
Exit block is now in UCFG, hence the increase of blocks

Reply:
and things will be optimized in after deserialization.